docs/conf.py

```diff
@@ -4,30 +4,21 @@
 # This file was created by sphinx-quickstart on Sun Nov  6 17:05:42 2016
 # and has been modified.
 #
-# Since Sherpa is tricky to build, in that it needs both a C and FORTRAN
-# compiler, which has been hard to set up on Read The Docs (although it
-# is getting close as of May 2018), the current approach is to mock
-# all the modules needed to create the documentation. That is, there is
-# no need to build Sherpa in order to build the documentation. This means
-# that we can get documentation for optional modules, but it does
-# require that the docstrings for functions in compiled modules are
+# Sherpa used to be tricky to build, but this is no longer the case.
+# We can now build it on ReadTheDocs and import the built version,
+# including the C modules.
+# However, in the past, the docstrings for functions in compiled modules were
 # added to the Python module rather than within the C API (as an example,
 # look at sherpa.utils which provides docstrings for the code in
-# sherpa.utils._utils.
-#
-# The minimum supported version of Sphinx is 1.8.
-#
-# The minimum requirements are:
-#    numpy  - since setup.py enforces this
-#    sphinx_rtd_theme
-#    sphinx_astropy
+# sherpa.utils._utils).
 #
 # The documentation can be built
 #   a) from the top level with 'python setup.py build_sphinx'
```

Comment:
So, if I try

```bash
pip install .
python setup.py build_sphinx
```

I get a whole bunch of errors. I know that support for this is going away

- see #1714

and my PR for setup.py changes 

- #1608 

cleans up the documentation, but misses this line.

Reply:
So we probably should just drop the "#   a) from the top level" line and change "#    b) from the docs/ .." to "#   .) from the docs/..."